torch/testing/_comparison.py

```diff
@@ -800,7 +800,16 @@ def _equalize_attributes(
             expected = expected.cpu()
 
         if actual.dtype != expected.dtype:
-            dtype = torch.promote_types(actual.dtype, expected.dtype)
+            actual_dtype = actual.dtype
+            expected_dtype = expected.dtype
+            # This is not sound in general, which is why promote_types doesn't
+            # allow it, but for easy testing, we're unlikely to get confused
+            # by large uint64 overflowing into negative int64
```

Comment:
We could add a check here that no value will overflow in this case since we don't care about the perf hit right?

Reply:
Yes, I suppose I could

Reply:
To do this, I have to implement gt_cpu. You want the binary ops? lol

Reply:
Huh,
I guess nice to have. So not required.